library/src/main/java/com/afollestad/aesthetic/Aesthetic.kt

```diff
@@ -534,9 +534,12 @@ class Aesthetic private constructor(private var context: Context?) {
     @SuppressLint("StaticFieldLeak")
     private var instance: Aesthetic? = null
 
+    @JvmStatic
+    fun attach(whereAmI: Context): Aesthetic = attach(whereAmI, null)
+
     /** Should be called before super.onCreate() in each Activity.  */
     @JvmStatic
-    fun attach(whereAmI: Context): Aesthetic {
+    fun attach(whereAmI: Context, inflationDelegate: InflationDelegate?): Aesthetic {
```

Comment:
The second parameter should have a default value, two methods aren't needed since we're using Kotlin